CondFormats/HcalObjects/interface/HFPhase1PMTData.h

```diff
@@ -26,7 +26,7 @@ class HFPhase1PMTData {
     ASYMM_MAX,    // Maximum allowed asymmetry
     N_PMT_CUTS
   };
-  typedef boost::array<boost::shared_ptr<AbsHcalFunctor>, N_PMT_CUTS> Cuts;
+  typedef boost::array<std::shared_ptr<AbsHcalFunctor>, N_PMT_CUTS> Cuts;
```

Comment:
@ggovi Is this change allowed? The record `HFPhase1PMTParamsRcd` is actively in use.

Reply:
Drawing attention of Igor @igv4321

Reply:
the change is allowed since this class does not uses the CMS standard persistency.

Reply:
Or better: from the DB side there are no objections since we don't maintain this 'private' persistency mechanism. Igor should comment and check if the change is compatible with their workflow

Reply:
Yes, indeed, I will need to bump up the class version inside the BOOST_CLASS_VERSION statement and update the "serialize" method so that it can read the previous version of the class as well. Does it have to be in this PR or can this change be factored out somehow?

Reply:
@igv4321 as I understand that this is not part of the standard CMS persistency, but just used privately, it is ok to go in a second PR IMO